packages/cli/src/commands/network/whitelist.test.ts

```diff
@@ -0,0 +1,22 @@
+import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { testLocally } from '../../test-utils/cliUtils'
+import Whitelist from './whitelist'
+
+process.env.NO_SYNCCHECK = 'true'
+
+// Lots of commands, sometimes times out
+jest.setTimeout(15000)
+
+testWithGanache('network:whitelist cmd', () => {
+  test('can print the whitelist', async () => {
+    const spy = jest.spyOn(console, 'log')
+
+    await testLocally(Whitelist, [])
+    expect(spy.mock.calls[0][0]).toMatchInlineSnapshot(`
```

Comment:
would it possible to test adapted one?

Reply:
Not easily, would require a couple things:

- either chain data to be used by ganache 
- or somehow make a new adapter + adapted token and deploy that on chain on startup

Reply:
you could mock the call to get the whitelist, that way we could cover the code responsible for formatting the adapted tokens